### ISSUE TITLE
No new test cases found. Please try amplifying another test case!

### ISSUE THREAD

**Issue by abolfazl-sh** [TestShiftProject/test-cube#3]
Hello,

I was using test-cube and found some problem.
The code I am trying to test is:
```
public static int tester(int a){
        if(a > 10){
            return --a;
        }else{
            return ++a;
        }
    }
```

and the Junit is:
```
import org.junit.Assert;
import org.junit.Test;

public class FirstTest {
    @Test
    public void firstTest() {

        Assert.assertEquals(Main.tester(12), 12);
    }
}
```

when I want to "Amplify 'firstTest()'", it doesn't work.
In the output "No new test cases found. Please try amplifying another test case!" was printed.
terminal output is :
```
log4j:ERROR setFile(null,true) call failed.
java.io.FileNotFoundException:  (Result too large)
```
(for detailed log see : https://paste.ubuntu.com/p/v9YcDYbPmV/)

java version "1.8.0_241"
IntelliJ IDEA 2021.2.2 (Ultimate Edition)
Thanks in advance.

**Comment by lacinoire** [comment]
Hey Abolfazl 😊

Thanks for trying out TestCube!
The error you posted is sadly in all outputs, looking at the detailed logs the actual error is further down and about the paths given.

```
(C:/Users/abolfazl/IdeaProjects/First\/src/main/java/)
```

Are you using Maven or Gradle? 🙂
Is there a sub-module in your project? (a folder between the root open in IntelliJ and the src/main folder)
TestCube can't do Gradle + sub-modules yet, work in progress these weeks 

Finally, are you using the published version from the Marketplace or building it yourself locally?
(depending on this I can propose different fixes 🙂)

**Comment by abolfazl-sh** [comment]
I wasn't use any build tools (non of Maven or Gradle).
I change it and use Gradle now (as you said in readme.md).
I am using the version from the marketplace. 
everything was going well but when the pop-up ask me to insert the maven dir I had inserted a wrong one.
Is there any way to change it?

**Comment by lacinoire** [comment]
In the seetings there is 🙂
Under "Tools" and "Test Cube Settings".

Though if you use gradle, the maven dir should not lead to it failing 🤔
What error are you having now? 😊

**Comment by abolfazl-sh** [comment]
Sorry I made a mistake. I mean I'm using maven now.
After changing Maven dir, I think everything is ok.
But the problem is that the tool doesn't make any new test case.
please check this : https://paste.ubuntu.com/p/6Ft3PJ6Z8D/ 
I think something wrong with my Junit code (you could find it in my first post).

**Comment by lacinoire** [comment]
From the log I see that building the project is working now 🙂
However, no test class named "FirstTest" is found...
I infer the class name from the test method the user tries to amplify, then DSpot searches for it in the test cases of the project. It seems like the test class is not where JUnit expects it? 🤔
Reads like valid JUnit 4 as far as I can see.

How is your file / project setup? 🙂

**Comment by abolfazl-sh** [comment]
I had made a directory and called it test.
I marked it as the test directory. When I run JUnit everything was well.
But I don't DSpot search the test folder under "src" directory.
By the way, I had changed JUnits directory to "src/test/java". 
After that something go wrong(the problem was about pom.xml and project structuring). This page helped : https://stackoverflow.com/a/17600801.
And now TestCube works like a charm.
Thanks for the time and hints.
